src/CustomElements/observe.js

```diff
@@ -263,16 +267,26 @@ var forEach = Array.prototype.forEach.call.bind(Array.prototype.forEach);
 
 // observe a node tree; bail if it's already being observed.
 function observe(inRoot) {
-  if (inRoot.__observer) {
+  
+  if (inRoot && inRoot.head && inRoot.head.__observer) {
     return;
   }
   // For each ShadowRoot, we create a new MutationObserver, so the root can be
   // garbage collected once all references to the `inRoot` node are gone.
   // Give the handler access to the root so that an 'in document' check can
   // be done.
+
+  // IE requires that you put an observer on child elements of the DOM or it will leak
+  // at this point inRoot == #document
   var observer = new MutationObserver(handler.bind(this, inRoot));
-  observer.observe(inRoot, {childList: true, subtree: true});
-  inRoot.__observer = observer;
+  observer.observe(inRoot.head, {childList: true, subtree: true});
```

Comment:
that my friend is what will causes a memory leak in IE; which is what it was before